Integrations/integration-IsItPhishing.yml

```diff
@@ -0,0 +1,166 @@
+description: Collaborative web service that provides validation on whether a URL is
+  a phishing page or not by analyzing the content of the webpage
+configuration:
+- display: Customer's name
+  name: name
+  defaultvalue: ""
+  type: 12
+  required: true
+- display: Customer's License
+  name: license
+  defaultvalue: ""
+  type: 12
+  required: true
+- display: Server URL
```

Comment:
provide an example in the display name (you may see other integrations that do so like wildfire)